climpred/constants.py

```diff
@@ -43,6 +43,9 @@
     PROBABILISTIC_HINDCAST_COMPARISONS + PROBABILISTIC_PM_COMPARISONS
 )
 
+# name for additional dimension in m2m comparison
+FORECAST_MEMBER_DIM = 'forecast_member'
```

Comment:
`M2E_MEMBER_DIM`. Can also add the `M2M_MEMBER_DIM` here for once we figure out these circular imports.